src/openrct2-ui/windows/Park.cpp

```diff
@@ -33,7 +33,6 @@
 
 static constexpr const rct_string_id WINDOW_TITLE = STR_STRINGID;
 static constexpr const int32_t WH = 224;
-static constexpr const int32_t WW = 230;
```

Comment:
NB: WW is no longer used; the MAIN_PARK_WIDGETS macro has recently been updated to take it as a parameter.